cblox_msgs/msg/MapPoseUpdate.msg

```diff
@@ -0,0 +1,5 @@
+# Message header
+Header header
+
+# Map header encapsulating the map's main properties
+cblox_msgs/MapHeader[] map_headers
```

Comment:
From the current name of this message it's not immediately clear that it contains pose updates for an array of submaps, instead of just a single one. What about calling this message `MapHeaderList` (clearest as to what it contains), `MapPoseList` (clearer in terms of its main use-case), or perhaps simply `MapPoseUpdates`?

Reply:
I think the pluralization, `MapPoseUpdates` is the best fit here. I will change it to that.